benefits/locale/en/LC_MESSAGES/django.po

```diff
@@ -364,13 +364,6 @@ msgstr ""
 msgid "Ride across %(count)s agencies (and growing!)"
 msgstr ""
 
-msgid "Please choose your transit provider:"
-msgstr ""
-
-#, python-format
-msgid "%(agency)s logo"
-msgstr ""
```

Comment:
Should we keep this for the new logo strip? (I note that it is not currently using a translation block for the logo alt text, though.)

Reply:
absolutely. it was a mistake that i omitted it in the logo strip.